test/models/user_test.rb

```diff
@@ -260,4 +260,11 @@ def user_opts
     assert_equal data[:zip], userInfo[:zip]
     assert_equal '/profile', redirect
   end
+
+  test 'military status accepts valid states' do
+    assert create(:user, military_status: 'spouse')
```

Comment:
Can you pls change these three to pass the three new constants, to support our validation?  i.e.

```ruby
assert create(:user, military_status: User::SPOUSE).valid?
```